content/docs/2.8/concepts/scaling-deployments.md

```diff
@@ -232,6 +232,29 @@ metadata:
 
 The presensce of this annotation will pause autoscaling no matter what number of replicas is provided. The above annotation will scale your current workload to 0 replicas and pause autoscaling. You can set the value of replicas for an object to be paused at to any arbitary number. To enable autoscaling again, simply remove the annotation from the `ScaledObject`definition.
 
+### `Activating` and `Scaling`
```

Comment:
This is still not renamed as suggested here: https://github.com/kedacore/keda-docs/pull/780#discussion_r899774978

I think it would be better to revert to your original name given it's more approachable to new people and less technical focussed.

It also feels a bit odd to see this in conceptual docs:
![image](https://user-images.githubusercontent.com/4345663/175871063-4d550c61-b3e1-4391-b478-f775c189dbb6.png)

Reply:
I didn't see it, omw

Reply:
No worries at all - That's what reviews are for - Thanks!

Reply:
reverted